### ISSUE TITLE
Jasypt 1.9.3 not working with OpenJDK version 262 and later: ERROR: java.lang.ExceptionInInitializerError

### ISSUE THREAD

**Issue by DSOANS** [jasypt/jasypt#44]
I have noticed that Jasypt 1.9,3 works fine with OpenJDK Redhat version **1.8.0.252-2.b09** but does NOT work with version 262 or 265 (latest).
I get the following error:
**ERROR: java.lang.ExceptionInInitializerError**

Any suggestions. There are vulnerability issues with OpenJDK version 252.

We are trying to upgrade the OpenJDK version but Jasypt is holding us back.

**Comment by ESM07** [comment]
Interesting, I get the same error when running the encrypt.bat command and I'm using JDK 1.8.0_261.

**Comment by DSOANS** [comment]
Yes, that is possible. The last working version I know of is 252. The next version I tested was with 262 as that was what was suggested based on the vulnerability. The latest is 265 which also does not work.

**Comment by DSOANS** [comment]
It appears the fix is to REMOVE the **com.ibm.icu4j jar (icu4j-3.4.4.jar)** file from the distribution - at least that seems to have worked for us.

**Comment by ESM07** [comment]
Great! that worked for me too. Thanks for sharing that!

**Comment by DSOANS** [comment]
Glad it worked for you. I will leave this issue as OPEN until the new IBM jar (version 67.1) is packaged.

**Comment by kien-truong** [comment]
https://github.com/jasypt/jasypt/commit/d384f9a755af2938bc142f7575365bee42ba5f22

This should be fixed a long time ago by this commit, but we'd have to wait till 1.9.4 to receive it.

**Comment by bornmw** [comment]
This is now a problem with Oracle Java as well
The latest 1.8.0_261 fails with the same exception, while 1.8.0_251 works perfectly fine
Please kindly produce a patch release as this is going to be a widespread issue

**Comment by ericnjogu** [comment]
I had the option of using `jenv` to switch to openjdk version 15 and that got rid of the error.
